src/game/GameController.cpp

```diff
@@ -58,15 +67,35 @@ GameController::~GameController()
 void GameController::run(sf::RenderWindow* window)
 {
     context->setWindow(window);
-
     window->setMouseCursorVisible(false);
 
+    context->changeMusic(currentMusicPath);
+
     while(window->isOpen())
     {
         window->clear();
 
-        currentController->render(context);
+        nextScreen = currentController->render(context);
 
         window->display();
+
+        if(nextScreen) {
+
+            nextMusicPath = musicFactory->getMusicPathById(
+                nextScreen
+            );
+
+            if(nextMusicPath != "" && currentMusicPath != nextMusicPath) {
```

Comment:
Do not return "" in any case and use templates on getMusicPathById() to return a boolean false if no music match